### ISSUE TITLE
Request Games

### ISSUE THREAD

**Issue by symbios24** [zerojay/RetroPie-Extra#252]
Hello, there as i mentioned some months before i have installed some extra games on retropie manual with the apt-get install command and they work great,but i don``t know how to do a pull request to your script, if you are interested the games list is as follows below:

openarena
wesnoth-1.11
minetest server (need compile)
minecraft server (needs compile)
assault cube server (needs compile)
ltris
lbreakout2
boswars
widelands
barrage
toppler
lgeneral
lmarbles
simutrans
tuxpuck
freecraft
berusky
tuxfootball
sopwith
nethack-x11
gnurobbo
enigma
freedroid

Thanks

**Comment by zerojay** [comment]
I'll take a look at them, thanks.

**Comment by Darknior** [comment]
This one is really cool and in 30 langages :)
http://wesnoth.org/
The problem will be to play mouse with pad like ever ... we really must add RETROPIE a real good mapper to map mouse to analog stick when it is needed.

**Comment by Darknior** [comment]
I think we have more than 7 8000 games possible on console an old computers on PI, with AMIGA, DOS, PC88 98, X68000, Atari ST ... I'm not sure people will install bad freeware to play them too.
I will not ... only the best one to complete console games, like Mario War, Rockbot engine, Solaris engine, Cave Story, if i found a good one Minecraft clone ...
There are also games like Quake, Doom, Heretic ... now are free to use, faster on native port than Dosbox.

For Minecraft there are so many clones ... i don't know what is the best one to play on PI and with ntwork option if possible to play with friends ?

https://wl.widelands.org/screenshots/
Really impressive too, and multi langage too :)

But many of these games have the mouse problem to play ... we/i must found a solution to have a good key mapper, easy to use on PI to add mouse on analogic stick .

**Comment by symbios24** [comment]
Mouse plays fine on all games that use it.

**Comment by Darknior** [comment]
I don't try the games for the moment. But I don't speak to use mouse. I speak about using joystick. Many people like me use Pi on TV in the living room and we use it like a console. We don't want to play with keyboard or mouse. Only joystick. It is easier. I know mouse is working fine on Linux games. I compile some other like you do.

**Comment by zerojay** [comment]
Joystick support for mouse driven games isn't a requirement for stuff to get submitted here though. I do note which ones require a mouse in the README.

**Comment by Darknior** [comment]
Yes for sure @zerojay you re right. I only want to say it can be interesting to think to found a solution for these games.

**Comment by Darknior** [comment]
And no translation, no hacks, no savestate, no filters, no bluetooth gamepad no HDMI ... nothing :(
I'm not fan of original consoles, i have them all when i was young, GG, Megadrive + switshs, SNES USA + Copier, NEC PCengine, NEC Turbo GT, PSX, Dreamcast, GB, DS, 3DS, Xbox, xBox360 ...
But i sold them ... i really prefer a little PI with all the games i love ready to play.

**Comment by symbios24** [comment]
There are packs that include almost everything that you mention-translation/hacks etc,all you need is an everdrive cart to load them, for example the snes games on retropie look really bad compared to the real thing even with a hdmi the resolution/graphics quality is terrible.thought it saves you from having all things consoles,the real thing is never replaceable.an emulator cannot reach it

**Comment by Darknior** [comment]
I know there are some diff but it is not important for me, the pleasure is 100% the same when i play with my children. I play from 30 years, i know all of that, i only make a choice :)
And it take me really less place in the living room ;) I love my choice i don't want to change.
Real console and everdrive will never replace all i write before for me. I forgot to speak about Boders too on 16/9 TV, Fast mod, rewind, ... nothing can be better than RETROARCH lol

**Comment by 8bitkid** [comment]
Thanks a lot a few things didn't start but on the whole I love the additions ROTT and the gamepad and srb2 have to be my favorites is their any chance of adding assault cube to this list?

**Comment by symbios24** [comment]
maybe in a couple of years

**Comment by levywinnikes** [comment]
Hi there, how i can install Berusky on retropie? Thanks

**Comment by symbios24** [comment]
hi its complicated try this

press f4 to go to command line type sudo apt-get update && sudo apt-get install berusky xorg

 cd /home/pi/RetroPie/roms/ports

sudo nano berusky.sh

type inside 

startx berusky or berusky 

press control+x and y to save

type emulationstation and press enter go in ports try it and let me know
